### PR TITLE
OP-1193 | Update ward Inventory

### PR DESCRIPTION
**Pull request by JantBogard**
See [OP-1193](https://openhospital.atlassian.net/browse/OP-1193)

[OP-1193]: https://openhospital.atlassian.net/browse/OP-1193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/java/org/isf/medicalinventory/service/MedicalInventoryRowIoOperation.java [hunk 1]
```diff
@@ -71,12 +71,26 @@ public void deleteMedicalInventoryRow(MedicalInventoryRow medicalInventoryRow) t
...
 	 * @return the list of {@link MedicalInventoryRow}s. It could be {@code empty}.
 	 * @throws OHServiceException
 	 */
 	public List<MedicalInventoryRow> getMedicalInventoryRowByInventoryId(int inventoryId) throws OHServiceException {
 		return repository.findByInventoryId(inventoryId);
 	}
+
+	/**
+	 * Return a list of {@link MedicalInventoryRow}s by Inventory Id and medical
+	 * code.
```
Comment (mwithi) [inline src/main/java/org/isf/medicalinventory/service/MedicalInventoryRowIoOperation.java:85]:
```suggestion
	 * Return a list of {@link MedicalInventoryRow}s by {@link Inventory} Id and {@link Medical} code.
```

**Review by mwithi — DISMISSED** [review]
LGTM. Thanks a lot!

File: src/main/java/org/isf/medicalinventory/model/MedicalInventory.java [hunk 2]
```diff
@@ -34,9 +37,6 @@
 import jakarta.persistence.Version;
 import jakarta.validation.constraints.NotNull;
 
-import org.isf.utils.db.Auditable;
-import org.springframework.data.jpa.domain.support.AuditingEntityListener;
-
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalinventory/model/MedicalInventory.java:39]: The import order was correct before you change; please the ide_settings directory for the correct ordering.

File: src/main/java/org/isf/medicalstock/manager/MovStockInsertingManager.java [hunk 3]
```diff
@@ -58,8 +59,10 @@ public MovStockInsertingManager(MedicalStockIoOperations medicalStockIoOperation
 	 * 
 	 * TODO: verify the need of checkReference param
 	 *
-	 * @param movement - the movement to validate
-	 * @param checkReference - if {@code true} it will use {@link #checkReferenceNumber(String) checkReferenceNumber}
+	 * @param movement       - the movement to validate
+	 * @param checkReference - if {@code true} it will use
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalstock/manager/MovStockInsertingManager.java:63]: So many of the changes are just formatting; and it looks like you aren't using the project default style.   And it looks like your line length is too short.   Please of all the formatting changes it makes it difficult to review the **real** changes.

**Pull request closed without merging**